docs/src/pages/company/careers/designer.md

```diff
@@ -0,0 +1,101 @@
+Today, countless teams and organizations rely on our open-source projects to build their design system.
+
+A couple of years ago, we started to expand our suite of products.
+We released [MUI X](/x/), a collection of advanced components; [MUI Design kits](/design-kits/), the MUI components available for the most popular design tools; and also host [Templates](/templates/), a set of pre-built UI kits.
+We've seen rapid growth with all of them (2-3x per year) and we have more to come.
+
+We are a fully distributed team, spread across multiple timezones.
+We work mainly asynchronously, relying mostly on written collaboration.
+Every contributor has the freedom to define how they work — the rest of us won't know what you're doing day-to-day unless you tell us.
+For additional details about the MUI team and culture, you can check our [careers](/careers/) and [about](/about/) pages and also our [public Handbook](https://mui-org.notion.site/Handbook-f086d47e10794d5e839aef9dc67f324b).
+
+## Why we're hiring
+
+Material-UI started as an open-source project.
+Over time, the audience and usage grew considerably, and some of the library's early contributors became full-time maintainers.
+A company was incorporated to support this growth, and more recently [was rebranded from Material-UI to MUI](/blog/material-ui-is-now-mui/).
+The new brand was fundamental to set the stage for MUI to become larger than Material Design, which was something we were already doing.
+Currently, we have five main projects that are not at all related to MD:
+
+- [MUI Core](/core/):
+  - [MUI Base](/customization/unstyled-components/): the unstyled version of the components.
+  - [MUI System](/system/basics/): our most recent take on styling for React components.
+  - [MUI Joy (working title)](https://github.com/mui-org/material-ui/discussions/29024): a second design system as an alternative to Material Design.
+- [MUI X](/x): as mentioned, a growing set of advanced components.
+  Today, the flagship is the [Data Grid](/components/data-grid/), a game-changing component for presenting large amounts of data, which integrates perfectly with MUI Core.
+- MUI Studio: a very recent endeavor aimed at exploring how our users can visually create apps 10x faster with the power of low-code and the flexibility of pro-code.
+
+We also know, especially due to [our annual developer survey](/blog/2020-developer-survey-results/), that design quality plays a huge part when developers are considering component library options.
+Therefore, we need to grow the design team to help us push these initiatives further.
+
+## About the role
+
+Some criteria for applying to this role:
+
+- Location: fully remote (preference for UTC-6 to UTC +3).
+- Type of work: full-time (as a contractor or employee, [depending primarily on location](https://mui-org.notion.site/Hiring-FAQ-64763b756ae44c37b47b081f98915501)).
```

Comment:
@mbrookes We were using this wording before:

```suggestion
- Type of work: full-time (as a contractor or employee, [depending on the case](https://mui-org.notion.site/Hiring-FAQ-64763b756ae44c37b47b081f98915501)).
```

Are you sure about the change? With Sam, we have https://www.letsdeel.com/ set up now. The page on Notion says:

> In specific occurrences, MUI can use an Employer of Record (EOR) (we use letsdeel.com/) to resolve specific issues but we haven't generalized this practice.

Reply:
> In specific occurrences, MUI can use an Employer of Record (EOR) (we use letsdeel.com/) to resolve specific issues but we haven't generalized this practice.

Hence it's primarily based on location (employee for France, contractor otherwise, occasional exception).

Reply:
@mbrookes OK, so this would be an accurate description of what we have done in the past. I agree.

However will it keep describing what we do in the future? The main problem I see is: are we missing out on better talents because the description? If we are, then I don't think that it's worth it. Are we? I don't know, but keeping it open ended could help us learn this.

Reply:
"depending on circumstances" will fix the grammar without changing the current meaning.

Reply:
Alright. I'm updating this, and we can move this discussion to a follow-up, this will unlock this PR to be merged.